web/src/main/ui/app/components/flows-new/edit-flow/mastering/matching/ui/add-match-option-dialog.component.ts

```diff
@@ -83,9 +83,7 @@ export class AddMatchOptionDialogComponent {
       return this.fb.array([this.createProp('')]);
     }
     const result = [];
-    this.data.option.propertyName.forEach(name => {
-      result.push(this.createProp(name))
-    })
+    result.push(this.createProp(this.data.option.propertyName));
```

Comment:
Reduce is working when only one property name is specified, but not when two or more property names are specified.

Reply:
@rjrudin - Please review now.